utils/note/getTextLength.ts

```
@@ -0,0 +1,39 @@
+import { convertDataForEditor } from './convertDataForEditor';
+
+import type { NoteContentTextLength } from '@/types/note.type';
+import type { StyledText } from '@blocknote/core';
+
+type CustomBlockType = {
+  id: string;
+  type: string;
+  props: unknown; // 사용하지 않기에 unknown
+  content: StyledText<{}>[];
+  children: unknown; // 사용하지 않기에 unknown
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)